source/developers-guide/index.html

```diff
@@ -12,7 +12,7 @@
     <li><a href="{{ site.url }}/developers-guide/shopware-5-search-bundle/">Shopware 5 SearchBundle - Easy product list selection</a></li>
     <li><a href="{{ site.url }}/developers-guide/shopware-5-core-service-extensions/">Shopware 5 core service extensions</a></li>
     <li><a href="{{ site.url }}/developers-guide/vagrant-phpstorm/">Vagrant and PHPStorm</a></li>
-    <li><a href="{{ site.url }}/developers-guide/shopware-5-performance/">Shopware 5 performance</a></li>
+    <li><a href="{{ site.url }}/developers-guide/shopware-5-performance-for-devs/">Shopware 5 performance guide for developers</a></li>
```

Comment:
Don't break existing URLs please. When needed include https://github.com/mavimo/sculpin-redirect-bundle and configure accordingly.